central/role/datastore/singleton.go

```diff
@@ -149,19 +150,27 @@ var defaultRoles = map[string]roleAttributes{
 	},
 }
 
+// TODO ROX-13888 when we migrate to WorkflowAdministration we can remove VulnerabilityReports and Role resources
 var vulnReportingDefaultRoles = map[string]roleAttributes{
```

Comment:
not your code, but I don't understand the point of var. 
Take it or ignore it: add this k-v directly to `defaultRoles`.